src/serac/infrastructure/mfem_configs.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2019-2024, Lawrence Livermore National Security, LLC and
```

Comment:
Should this go into `serac_config.hpp` instead?

Reply:
Agreed